frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-body-independent.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { Grid, GridItem } from '@patternfly/react-core';
+import { Humanize } from '@console/internal/components/utils';
+import { K8sKind } from '@console/internal/module/k8s';
+import { addAvailable, StackDataPoint } from './utils';
+import { BreakdownChart } from './breakdown-chart';
+import { BreakdownChartLoading } from './breakdown-loading';
+import { TotalCapacityBody } from './breakdown-capacity';
+
+export const BreakdownCardBody: React.FC<BreakdownBodyProps> = ({
+  top5MetricsStats,
+  metricTotal,
+  metricModel,
+  humanize,
+  isLoading,
+  hasLoadError,
+  ocsVersion = '',
+}) => {
+  if (isLoading && !hasLoadError) {
+    return <BreakdownChartLoading />;
+  }
+  if (!top5MetricsStats.length || hasLoadError) {
+    return <div className="text-secondary">Not available</div>;
+  }
+  if (metricTotal === '0') {
+    return <div className="text-secondary">Not enough usage data</div>;
+  }
+
+  const usedCapacity = `${humanize(metricTotal).string} used`;
+
+  const chartData = addAvailable(top5MetricsStats, null, null, metricTotal, humanize);
+
+  const legends = chartData.map((d: StackDataPoint) => ({
+    name: [d.name, d.label],
+    labels: { fill: d.color },
+    symbol: { fill: d.fill },
+    link: d.link,
+    ns: d.ns,
+  }));
+
+  return (
+    <Grid>
+      <GridItem span={4}>
+        <TotalCapacityBody value={usedCapacity} />
+      </GridItem>
+      <GridItem span={4} />
+      <GridItem span={12}>
+        <BreakdownChart
+          data={chartData}
+          legends={legends}
+          metricModel={metricModel}
+          ocsVersion={ocsVersion}
+        />
+      </GridItem>
+    </Grid>
+  );
+};
+
+type BreakdownBodyProps = {
+  isLoading: boolean;
+  hasLoadError: boolean;
+  metricTotal: string;
+  top5MetricsStats: StackDataPoint[];
+  metricModel: K8sKind;
+  humanize: Humanize;
+  ocsVersion?: string;
+};
```

Comment:
You can  actually reuse the existing component. You dont need `capacityTotal` for independent mode, and that is already optional for BreakdownBody.
Also just like noobaa, capacity total === metricTotal here.

Refer: https://github.com/openshift/console/blob/f051348ca4db91bbccdf03fde9bc074975348d6d/frontend/packages/noobaa-storage-plugin/src/components/capacity-breakdown/capacity-breakdown-card.tsx#L111-L119

Reply:
@afreen23 the problem lies with used capacity. I couldn't see a workaround it. Do you have any suggestions around it?